README.md

```diff
@@ -670,24 +721,25 @@ make[1]: Leaving directory '/home/developer/src/test'
 developer:test > 
 ```
 
-There is no difference between the files and the test was successful. To see how `check-diff` detects unintended changes to the configuration break the `loopback-template.xml` file on purpose. Open the `loopback/templates/loopback-template.xml` file in the editor and remove the line 21:
+There is no difference between the files and the test was successful. To see how `check-diff` detects unintended changes to the configuration break the `loopback-template.xml` file on purpose. Open the `~/src/loopback/templates/loopback-template.xml` file in the editor and remove the line 21:
 ```
         <mask>255.255.255.255</mask>
 ```
 
 This will create the bgp loopback interface without the mask. Copy changed file to NSO running directory:
 ```
-cp loopback/templates/loopback-template.xml /nso/run/packages/loopback/templates/
+cp ~/src/loopback/templates/loopback-template.xml $NCS_RUN_DIR/packages/loopback/templates/
 ```
 Output:
 ```
-developer:src > cp loopback/templates/loopback-template.xml /nso/run/packages/loopback/templates/
+developer:src > cp ~/src/loopback/templates/loopback-template.xml $NCS_RUN_DIR/packages/loopback/templates/
 ```
 
 Enter NSO CLI and redeploy the loopback package:
```

Comment:
can you share the reason why you need to redeploy? I thought you need to reload the package. it will be good for users to know the reason.

Reply:
Redeploy is enough when only Python code or template is changed for a service package. It is faster than full packages reload since there is no schema reload.